client/src/components/ratingsReviews/RatingBreakdown.jsx

```diff
@@ -2,66 +2,64 @@ import React from 'react';
 import RatingProgress from './RatingProgress.jsx';
 import StarRating from '../shared/StarRating.jsx';
 
-
 const RatingBreakdown = (props) => {
-  const { reviews, ratings, recommended } = props;
+  const { ratings, recommended, handleRatingProgressFilter } = props;
   const stars = [1, 1, 1, .8, 0];
   let key = 0;
 
-  const oneBar = parseInt(ratings[1]) || 0;
-  const oneStar = oneBar * 1 || 0;
-  const twoBar = parseInt(ratings[2]) || 0;
-  const twoStar = twoBar * 2 || 0;
-  const threeBar = parseInt(ratings[3]) || 0;
-  const threeStar = threeBar * 3 || 0;
-  const fourBar = parseInt(ratings[4]) || 0;
-  const fourStar = fourBar * 4 || 0;
-  const fiveBar = parseInt(ratings[5]) || 0;
-  const fiveStar = fiveBar * 5 || 0;
 
-  const sum = oneStar + twoStar + threeStar + fourStar + fiveStar;
-  const sumBar = oneBar + twoBar + threeBar + fourBar + fiveBar;
-  const averageRating = sum / sumBar;
+  if (ratings !== null && ratings) {
```

Comment:
Isn't this redundant? I can't think of a case where `if (ratings)` would be insufficient.

Reply:
It is redundant, I've updated it with your suggestion